app/plugins/auth-refresh.js

```diff
@@ -16,13 +17,13 @@ module.exports = {
         }
 
         const currentToken = request.state[AUTH_COOKIE_NAME]
-        const refreshToken = request.state[AUTH_REFRESH_COOKIE_NAME]
+        const refreshToken = request.yar.get(REFRESH_TOKEN)
```

Comment:
This logic needs to be changed, but for now accommodating that the token is now in the refresh cache.